runtime/doc/tabpage.txt

```diff
@@ -61,16 +61,25 @@ In the GUI tab pages line you can use the right mouse button to open menu.
 :[count]tabe[dit]				*:tabe* *:tabedit* *:tabnew*
 :[count]tabnew
 		Open a new tab page with an empty window, after the current
-		tab page.  For [count] see |:tab| below.
+		tab page.  If [count] is given the new tab page appears after 
+		the tabpage [count] otherwise the new tab page will appear
+		after the current one. >
+		    :tabnew     " opens tabpage after the current one
```

Comment:
I'm neutral on this one. Perhaps it should be included in a further PR to clean up the whole thing.

Not sure how that would impact merging changes to the documentation from vim, though.